modules/nw-kuryr-migration.adoc

```diff
@@ -0,0 +1,338 @@
+----
++
+The name of the Multus pods is in the form of `multus-<xxxxx>`, where `<xxxxx>` is a random sequence of letters. It might take several moments for the pods to restart.
++
+.Example output
+[source,text]
+----
+Waiting for daemon set "multus" rollout to finish: 1 out of 6 new pods have been updated...
+...
+Waiting for daemon set "multus" rollout to finish: 5 of 6 updated pods are available...
+daemon set "multus" successfully rolled out
+----
+
+. To complete the migration, reboot each node in your cluster. For example, you can use a bash script similar to the following example. The script assumes that you can connect to each host by using `ssh` and that you have configured `sudo` to not prompt for a password.
++
+[source,bash]
+----
+#!/bin/bash
+
+for ip in $(oc get nodes  -o jsonpath='{.items[*].status.addresses[?(@.type=="InternalIP")].address}')
+do
+   echo "reboot node $ip"
+   ssh -o StrictHostKeyChecking=no core@$ip sudo shutdown -r -t 3
+done
+----
++
+[NOTE]
+====
+If ssh access is not available, you could alternatively use `openstack` command
+to do so:
+[source,terminal]
+----
+$ CLUSTERID=$(oc get infrastructure.config.openshift.io cluster -o=jsonpath='{.status.infrastructureName}')
+$ for name in $(openstack server list --name ${CLUSTERID}\* -f value -c Name); do openstack server reboot $name; done
+----
+It might also be able to reboot each node through the management portal for
+your infrastructure provider. Otherwise contact appropriate authority for
+either gaining access to the virtual machines through ssh or management
+portal/OpenStack client.
+====
+
+. Confirm that the migration succeeded, and then remove the migration resources:
+
+.. To confirm that the network plugin is OVN-Kubernetes, enter the following command.
++
+[source,terminal]
+----
+$ oc get network.config/cluster -o jsonpath='{.status.networkType}{"\n"}'
+----
++
+ The value of `status.networkType` must be `OVNKubernetes`.
+
+.. To confirm that the cluster nodes are in the `Ready` state, enter the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
+
+.. To confirm that your pods are not in an error state, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods --all-namespaces -o wide --sort-by='{.spec.nodeName}'
+----
++
+If pods on a node are in an error state, reboot that node.
+
+.. To confirm that all of the cluster Operators are not in an abnormal state, enter the following command:
++
+[source,terminal]
+----
+$ oc get co
+----
++
+The status of every cluster Operator must be the following: `AVAILABLE="True"`, `PROGRESSING="False"`, `DEGRADED="False"`. If a cluster Operator is not available or degraded, check the logs for the cluster Operator for more information.
+
+IMPORTANT: Do not proceed if the any of the previous verification steps indicate errors, but note, that there can be pods with Terminating state due to the finalzers which will be removed during cleanup step.
```

Comment:
Added note about pods in Terminating state.

Reply:
```suggestion
[IMPORTANT]
====
Do not proceed if the any of the previous verification steps indicate errors.

You might encounter pods that have a  `Terminating` state due to finalizers that are removed during clean up. They are not an error indication.  
====
```

Is this an accurate treatment?